### DISCUSSION TITLE
Add custom icon to the Bootstrap file

### DISCUSSION THREAD

**Discussion by paradoxx42** [rfvgyhn/min-ed-launcher#130] [Ideas]
![1](https://github.com/rfvgyhn/min-ed-launcher/assets/149199215/3f45aeb5-3817-4941-8d34-0098e06288e1)
When launching with Epic, the desktop shortcut is just a default white "paper" icon because it links to the renamed MinEdLauncher.bootstrap.exe file which doesn't include an icon. My idea is to create a custom icon based on the original launcher's icon in order to improve the user experience (maybe just put some text "Min ED" over the icon).

**Comment by rfvgyhn** [comment]
Good idea. [Icon added](https://github.com/rfvgyhn/min-ed-launcher/blob/master/resources/min-ed-launcher.svg) and will be in the next release.

Text can be hard to read when the image is small so I opted to not have any text.
